item_selection/book recommenders/goodreads_1k_data.csv

```diff
@@ -0,0 +1,1005 @@
044,3692,35750,1506,31591,11551,5415,33699,33933,35054,19377,32281,18261,23504,32503,13797,35806,40859,37169,10500,42007,42148,7352,43128,30253,10887,24337,40923,30296,4191,19173,1494,29920,6956,24192,40199,29589,7859,37572,30702,14605,22242,19279,23693,253,15549,12016,11006,25936,24605,38597,35519,19618,26934,41702,6339,25689,11077,13990,27421,2817,1465,2949,24010,15047,1226,25623,29446,24569,40361,15600,17315,8148,3531,24979,3842,45549,31260,5202,11047,16271,33807,14990,25737,431,35244,8012,22515,16634,16435,41426,44156,16374,5513,17043,9717,32522,10382,29622,32506,38529,19352,12770,40359,31878,2860,26968,24729,22898,30371,22399,7186,17645,18853,4799,22286,15905,11251,23506,10973,22135,13060,41089,7286,5886,2283,4906,12240,28597,19673,5332,10920,24904,21402,29619,29153,1946,9915,11137,33993,30065,22472,9304,7869,39392,2873,11741,34440,39690,45253,1368,28462,966,4248,29470,20677,9355,34938,20468,29526,39286,15867,5848,8552,35908,11934,13872,1428,41317,39795,30659,41505,24888,25946,37134,30271,30977,38436,12652,10713,25426,36535,22364,3782,18926,13622,6176,19331,3311,18628,14769,31107,25033,13994,21838,36333,18638,3075,4693,35711,19631,31822,39399,15624,32533,32351,45121,44039,9479,29734,33692,35056,8712,6221,98,41704,13140,24060,37612,36717,15124,698,30774,409,24727,45106,11326,11184,19689,4117,12783,31092,39580,25853,367,21673,3693,22739,29584,43486,40342,28344,10570,14176,8853,44439,12537,36076,18266,23522,33544,41840,16682,13142,30104,1814,147,41664,20517,4060,2535,30536,15466,16179,28664,11933,22445,9156,44184,36568,3659,23968,43268,39793,14570,9300,38700,25103,16806,14243,17639,33295,6556,45267,42278,35332,36706,11225,6745,3509,17513,42993,29093,33774,27693,4658,18947,13650,11159,44617,33137,19910,23475,27415,16245,45574,14430,8246,37476,2883,18172,35111,41463,24119,16184,26593,15880,5168,34771,30122,39872,42390,6217,14905,6619,8750,30165,29788,32349,7896,9816,4232,2998,16322,30300,37744,28418,36946,7337,22417,13836,32656,29998,15054,45220,19393,44089,12840,27769,5062,5480,32070,32813,4515,45114,21362,8494,1252,19302,16790,16331,27737,668,31894,32741,40459,45205,27419,5691,24686,9107,26723,3431,5911,44187,19888,13211,9997,18646,25200,25937,45382,42647,9005,30417,10880,8879,29394,38066,3734,1511,3657,16685,16280,30992,40271,16,19397,24581,41649,40911,42694,44936,1547,20876,6159,31049,20949,10914,3631,35258,30243,19807,36639,45438,27822,12645,32916,13649,15474,10471,29084,3869,8746,17216,16192,34816,35335,19672,16921,11621,31270,11695,4331,14790,14712,9806,17412,36368,9567,18135,20127,6216,32307,20264,15216,41979,38724,1850,45159,6174,38769,32097,42597,42862,30160,25564,12550,14808,39784,11376,39685,33836,37195,37552,10427,955,21107,6615,28302,3758,14930,11441,4315,20935,2353,3519,30482,30382,15195,5130,40159,24249,5665,7677,43918,13493,9501,24865,2528,33356,2552,25934,8661,11389,20895,33026,35409,3750,18837,21746,27647,27695,42898,36952,12495,11031,17675,30385,25112,5216,34768,3536,15425,21290,18556,38414,15075,18828,6298,93,44821,11036,2315
```

Comment:
Do these IDs correspond to those in the original Goodreads data?

Reply:
I think yes. These IDs are sampled from the column `book_id` in books_processed.csv`. 

Maybe these IDs are not meaningful to users? the IDs seem to come from nowhere. I feel these IDs can be any ID though, as long as the text content and labels follow the same order. So maybe we can create new IDs like 0 ~ 999 instead of using current IDs from source file.

Reply:
Yes if these don't correspond to IDs in the original data then just using 0 to 999 (or 1 to 1000) would be better I think.